src/status_im/ui2/screens/chat/composer/view.cljs

```diff
@@ -243,20 +241,27 @@
                                                             set-bg-opacity)]
               (quo.react/effect!
                #(do
-                  (when (and @keyboard-was-shown? (not keyboard-shown))
-                    (swap! context assoc :state :min)
-                    (set-bg-opacity 0))
-                  (when (and blank-composer? (not (seq images)) (not edit))
-                    (clean-and-minimize-composer-fn false))
-                  (when (seq images)
-                    (input/show-send refs))
-                  (reset! keyboard-was-shown? keyboard-shown)
-                  (if (#{:max :custom-chat-unavailable} (:state @context))
-                    (set-bg-opacity 1)
-                    (set-bg-opacity 0))
-                  (reanimated/set-shared-value translate-y (reanimated/with-timing (- y)))
-                  (reanimated/set-shared-value shared-height
-                                               (reanimated/with-timing (min y max-height)))))
+                  (let [edited? (-> (rf/sub [:chat/inputs])
```

Comment:
This is a tad too difficult to understand to me, besides @ibrkhalil, you could consider having another subscription to keep the UI dumb.

```clojure
(let [input-text (get-in (rf/sub [:chat/inputs])
                         [chat-id :input-text])
      edited?    (not= input-text (-> edit :content :text))])
```